mappings/net/minecraft/entity/passive/PandaEntity.mapping

```diff
@@ -19,6 +19,12 @@ CLASS aqs net/minecraft/entity/passive/PandaEntity
 		METHOD c isRecessive ()Z
 	CLASS aqs$c PandaFleeGoal
 	CLASS aqs$d PandaMateGoal
+		FIELD d CLOSE_PLAYER_PREDICATE Lapp;
+		FIELD e pandaEntity Laqs;
```

Comment:
`owner`. Moreover, I usually don't suggest mapping these fields as they can be leftovers from decompiler errors (e.g. In original code, it may be `PandaEntity.this` in an inner class, but in bytecode it is actually a field and might have been obfuscated by proguard)

Reply:
Should I unmap all inner class `T.this`-type statements? Also `owner` doesn't work here, it's already used by the class it's extending (After changing `AnimalMateGoal` to use `owner`)